### DISCUSSION TITLE
how to hide control when provide for threejs CanvasTexture ?

### DISCUSSION THREAD

**Discussion by 826327700** [fabricjs/fabric.js#7973] [Q&A] [answered]
when I use fabric canvas create CanvasTexture in threejs, I hope hide control. 
I try this code:
```
drawControls(ctx: any) {
        var activeObject = this._activeObject;

        if (activeObject) {
            //@ts-ignore
            activeObject._renderControls(this.upperCanvasEl.getContext("2d"));
        }
    }
```
but this control repeat draw in fabric canvas,like this
![image](https://user-images.githubusercontent.com/23014381/170417533-6074f0a3-3bcf-4f27-a0c7-bd484bf3f3ae.png)

**Comment by ShaMan123** [comment, marked as answer]
hasControls property

**Comment by ShaMan123** [comment]
Could you provide a working example of integrating fabric and Three?
I would like to put that in the website's demos
